ui/src/components/NewExperimentNext/data/types.ts

```diff
@@ -534,14 +541,430 @@ const data: Record<Kind, Definition> = {
   },
 }
 
-const AwsChaosCommonSchema = Yup.object({
+const networkPhysicCommon: Spec = {
+  correlation: {
+    field: 'text',
+    label: 'Correlation',
+    value: '',
+    helperText: 'The correlation of corrupt',
+  },
+  device: {
+    field: 'text',
+    label: 'Device',
+    value: '',
+    helperText: 'Affected device, e.g., eth0',
+  },
+  hostname: {
+    field: 'text',
+    label: 'Hostname',
+    value: '',
+    helperText: 'Specify the hostname',
+  },
+  'ip-address': {
+    field: 'text',
+    label: 'IP Address',
+    value: '',
+    helperText: 'Specify the IP address',
+  },
+  'ip-protocol': {
+    field: 'select',
+    items: ['tcp', 'udp', 'icmp', 'all'],
+    label: 'IP Protocol',
+    value: 'all',
+    helperText: 'Specify the IP protocol',
+  },
+  sourceport: {
+    field: 'text',
+    label: 'Source Port',
+    value: '',
+    helperText: 'The source port, split by ,',
+    if: {
+      key: 'ip-protocol',
+      equal: ['tcp', 'udp'],
+    },
+  },
+  'egress-port': {
+    field: 'text',
+    label: 'Egress Port',
+    value: '',
+    helperText: 'The egress port, split by ,',
+  },
+  percent: {
+    field: 'text',
+    label: 'Percent',
+    value: '1',
+    helperText: 'Percentage of network packet duplication',
+  },
+}
+
+const diskPhysicCommon: Spec = {
+  size: {
+    field: 'text',
+    label: 'Size',
+    value: '',
+    helperText:
+      'The supported formats of the size are: c(=1), w(=2), kB(=1000), K(=1024), MB(=1024), M(=1024x1024), GB and so on.',
+  },
+  path: {
+    field: 'text',
+    label: 'Path',
+    value: '',
+    helperText: 'Specify the file path of the data to be read/written',
```

Comment:
@WangXiangUSTC Dashboard could only get events which is like `HTTP status is not ok` now, If I want to see more information about this error, I need to check logs on the physical machine. I wonder if we could return the error information to chaos-controller-manager, and then we can see more detailed events from dashboard.